misc/testerina/modules/testerina-runtime/src/main/java/org/ballerinalang/test/runtime/BTestMain.java

```diff
@@ -108,13 +112,12 @@ public static void main(String[] args) throws IOException {
                 } else {
                     br = Files.newBufferedReader(testSuiteJsonPath, StandardCharsets.UTF_8);
                 }
-                Gson gson = new Gson();
-                Map<String, TestSuite> testSuiteMap = gson.fromJson(br,
-                        new TypeToken<Map<String, TestSuite>>() { }.getType());
+                Object jsonObj = JsonUtils.parse(br, JsonUtils.NonStringValueProcessingMode.FROM_JSON_STRING);
+                BMap<BString, Object> testSuiteMap = (BMap<BString, Object>) jsonObj;
```

Comment:
```suggestion
                BMap<BString, Object> testSuiteMap = (BMap<BString, Object>) JsonUtils.parse(br,
                JsonUtils.NonStringValueProcessingMode.FROM_JSON_STRING);
```
Because we only use 'jsonObj` in one place.

Reply:
Fixed in f7de7e58700109d16648e2c12f8c42cf3190286d